.github/workflows/ci.yml

```diff
@@ -16,24 +16,22 @@ jobs:
       - name: Docker login
         uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc #v2
         with:
-          # registry: techplatform0scaffolding0dev0acr.azurecr.io
-          # username: ${{ secrets.WORKLEAP_ACR_USERNAME }}
-          # password: ${{ secrets.WORKLEAP_ACR_PASSWORD }}
-          username: ${{ secrets.ERIC_ACR_USERNAME }}
-          password: ${{ secrets.ERIC_ACR_PASSWORD }}
+          registry: techplatform0scaffolding0dev0acr.azurecr.io
+          username: ${{ secrets.WORKLEAP_ACR_USERNAME }}
+          password: ${{ secrets.WORKLEAP_ACR_PASSWORD }}
+
 
       - name: Docker metadata
         id: meta
         uses: docker/metadata-action@818d4b7b91585d195f67373fd9cb0332e31a7175 #v4
         with:
-          # images: techplatform0scaffolding0dev0acr.azurecr.io/eventgrid-emulator
-          images: ericdecarufel/eventgrid-emulator-test
+          images: techplatform0scaffolding0dev0acr.azurecr.io/eventgrid-emulator
           
       - name: Docker build and push
         uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825 #v4
         with:
           context: ./src
           file: ./src/EventGridEmulator/Dockerfile
-          push: false
+          push: true
```

Comment:
Can you add a comment stating that this is temporary? Because this workflow is not supposed to publish images. That's the purpose of the publish.yml workflow, which should be triggered when creating releases.

Reply:
Same thing for the registry and image name.

The goal of this workflow was simply to validate that we could build a Docker image